metrics/models.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from topics.models import Article, Topic
+
+
+class Metric(models.Model):
```

Comment:
Why do we call them metrics? A metric to me should be for things like "Amount of bias", as in the resulting statistic computed from the moderations left by users. This class is more like a reaction/comment/instance of moderation IMO.